src/index.ts

```diff
@@ -0,0 +1,353 @@
+import yargs from "yargs/yargs";
+import { Wallet, providers, ethers } from "ethers";
+import { config } from "dotenv";
+import { Argv } from "yargs";
+import { IExchange__factory } from "./generated/factory/IExchange__factory";
+import { IERC20__factory } from "./generated/factory/IERC20__factory";
+import { LiquidationBotApi__factory } from "./generated/factory/LiquidationBotApi__factory";
+
+const SLICE_SIZE = 1000;
+
+export function checkDefined<T>(
+  val: T | null | undefined,
+  message = "Should be defined"
+): T {
+  if (val === null || val === undefined) {
+    throw new Error(message);
+  }
+  return val;
+}
+
+const loadAccount = function (networkId: string, accountNumber: number) {
+  if (accountNumber < 0 || accountNumber >= 200) {
+    throw new Error("Invalid account number: " + accountNumber);
+  }
+  config();
+
+  const mnemonic = checkDefined(
+    process.env[`${networkId}_MNEMONIC`],
+    `Missing ${networkId}_MNEMONIC in your .env file, see README.md`
+  );
+
+  return Wallet.fromMnemonic(
+    mnemonic,
+    `m/44'/60'/0'/0/${accountNumber}`
+  ).connect(getProvider(networkId));
+};
+
+const getProvider = function (networkId: string) {
+  const url = checkDefined(
+    process.env[`${networkId}_RPC_URL`],
+    `Missing ${networkId}_RPC_URL in your .env file, see README.md`
+  );
+  const chainId = checkDefined(
+    process.env[`${networkId}_CHAINID`],
+    `Missing ${networkId}_CHAINID in your .env file, see README.md`
+  );
+
+  return new providers.JsonRpcProvider(url, {
+    name: "test",
+    chainId: Number(chainId),
+  });
+};
+
+const getStandardParams = (argv: any) => {
+  return {
+    accountNumber: argv.accountNumber as number,
+    networkId: (argv.networkId as string).toUpperCase(),
+    exchangeAddress: (argv.exchangeAddress as string).toLowerCase(),
+  };
+};
+
+const main = async () => {
+  await yargs(process.argv.slice(2))
+    .option("networkId", {
+      alias: "n",
+      describe: "network where this will be run",
+      type: "string",
+      default: "localhost",
+    })
+    .option("accountNumber", {
+      alias: "x",
+      describe: "fs test account",
+      type: "number",
+      default: 0,
+    })
+    .option("exchangeAddress", {
+      alias: "e",
+      describe: "exchange address",
+      type: "string",
```

Comment:
We may also add `.coerse` to make the library do the validation and conversion:

```javascript
    .option("exchangeAddress", {
      alias: "e",
      descript: "exchange address",
      type: "string",
      coerse: (val) => ethers.utils.getAddress(val),
    })
```

https://github.com/yargs/yargs/blob/master/docs/api.md#coercekey-fn
https://docs.ethers.io/v5/api/utils/address/#utils-getAddress